### ISSUE TITLE
Article details: Save / Reblog / Comment / Like icons and text are not aligned and visible

### ISSUE THREAD

**Issue by antonis** [wordpress-mobile/WordPress-iOS#21835]
### Expected behavior
On small screen or with large text/display configuration the Save / Reblog / Comment / Like actions and icons are aligned and fully visible similar to how the tab bar behaves

### Actual behavior
On small screen or with large text/display configuration the Save / Reblog / Comment / Like actions and icons are not aligned and fully visible

### Steps to reproduce the behavior
1. Set a large font size or the large text display zoom
2. Browse at an article detail
3. Notice that the Save / Reblog / Comment / Like actions and icons are not aligned and not fully visible

|Article 1|Article 2|
|---|---|
|![IMG_0209](https://github.com/wordpress-mobile/WordPress-iOS/assets/304044/e8ee9e8a-ea85-4121-856a-0a4c7def83d5)|![IMG_0214](https://github.com/wordpress-mobile/WordPress-iOS/assets/304044/f872347e-416f-4a9b-bc88-578aa5b93406)|

**Comment by dvdchr** [comment]
Hi @antonis 👋🏼 , thank you for the report! This _should_ be addressed via #21833, where I restricted the toolbar labels to one line. It should be testable in the next beta! I'll keep the issue open in case you want to verify this again once the new beta is ready. 🙂

**Comment by antonis** [comment]
Hey @dvdchr 👋 
Thank you for following up on this 🙇 
I've retested with the CI build from https://github.com/wordpress-mobile/WordPress-iOS/pull/21833 and 23.5.0.2 from TestFlight and I'm still able to reproduce the issue on my  iPhone SE 2nd gen. either by setting the display zoom to large text or by setting the text size at 6/7. Given that [my device has a small screen](https://en.wikipedia.org/wiki/IPhone_SE_(2nd_generation)#Display) with the current standards this might be an edge case.

|Display Zoom|Text size|Post details|
|---|---|---|
|![IMG_0216](https://github.com/wordpress-mobile/WordPress-iOS/assets/304044/fef18178-1e93-4254-b8f1-38af258dc526)|![IMG_0218](https://github.com/wordpress-mobile/WordPress-iOS/assets/304044/b4b76672-22d8-40d9-84ba-de16156f4b18)|![IMG_0217](https://github.com/wordpress-mobile/WordPress-iOS/assets/304044/e5e4ea6a-a187-4525-b908-e34dbb21d433)|

**Comment by dvdchr** [comment]
Thanks @antonis ! I'll take a closer look.

**Comment by dvdchr** [comment]
Closed via #21870.
